internal/cmd/asyncapi/command_export.go

```diff
@@ -108,12 +108,22 @@ func (c *command) export(cmd *cobra.Command, _ []string) error {
 	}
 	channelCount := 0
 
-	topicsSpecified := types.NewSet(flags.topics...)
+	topicsSpecified := types.NewSet()
+	var prefixesSpecified []string
+	// Split user-specified topics into topics/topic prefixes
+	for _, userTopic := range flags.topics {
+		if userTopic[len(userTopic)-1:] == "*" {
```

Comment:
can we add a length check here to avoid panic? in case user input "" by mistake

Reply:
https://pkg.go.dev/strings#HasSuffix has this check built-in!